Create ui-c8y-1024-7-1-fixed-the-issue-with-adding-a-custom-column-in-the-data.md (#4954)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1024-7-1-fixed-the-issue-with-adding-a-custom-column-in-the-data.md

---
date: ""

> title: Custom columns in data grids now work correctly

Reviewer: ```suggestion
title: Custom columns using Digital Twin Manager properties now work correctly in data grids
```

> title: Custom columns using Digital Twin Manager properties now work correctly in data grids

Reviewer 2: ```suggestion
title: Custom columns using Digital Twin Manager properties now display correctly in data grids
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-66948
version: 1024.7.1
---

> Custom columns based on Digital Twin Manager properties added to a data grid were silently dropped and never appeared. The selected columns now display as expected.

Reviewer 2: ```suggestion
Custom columns based on Digital Twin Manager properties that were added to a data grid were silently dropped and never appeared. The selected columns now display as expected.
```

> The data grid feature allows you to customize which columns display your data, helping you focus on the information most relevant to your work. Previously, when you attempted to add a custom column to a data grid, the operation would fail and the column would not be added. Now, adding custom columns works as expected, and your selected columns display correctly in the data grid. This fix ensures that you can fully customize your data grid views without encountering errors, improving your ability to organize and view your data efficiently.

Reviewer: ```suggestion
Custom columns based on Digital Twin Manager properties added to a data grid were silently dropped and never appeared. The selected columns now display as expected.
```
